packages/core/src/components/editable-text/editableText.tsx

```diff
@@ -6,10 +6,8 @@
  */
 
 import * as classNames from "classnames";
-import * as PureRender from "pure-render-decorator";
 import * as React from "react";
-
-import { AbstractComponent } from "../../common/abstractComponent";
+import { AbstractPureComponent } from "../../common/abstractPureComponent";
```

Comment:
leave the blank line above. it separates import groups so they can be alphabetized separately.